bc_obps/registration/api/v2/_user_operators/_user_operator_id/update_status.py

```diff
@@ -23,7 +23,7 @@
     description="""Updates the status of a user operator by its ID.
     If the status is updated to 'APPROVED' or 'DECLINED', the user operator is verified with the current timestamp and the admin who performed the action.
     An email notification is sent based on the updated status.""",
-    auth=authorize("authorized_irc_user_and_industry_admin_user"),
```

Comment:
Since I added `cas_view_only` to `authorized_irc_user`, any non-GET endpoints that used it had to be updated